Files/Strings/de-DE/Resources.resw

```diff
@@ -1002,94 +1002,4 @@
-    <value>Hierher verschieben</value>
-  </data>
-  <data name="DrivesWidgetDescription.Text" xml:space="preserve">
-    <value>Laufwerke</value>
-  </data>
-  <data name="SettingsWidgetsShowLibraryCards.AutomationProperties.Name" xml:space="preserve">
-    <value>Bibliotheken auf Startseite anzeigen</value>
-  </data>
-  <data name="SettingsWidgetsShowLibraryCards.Header" xml:space="preserve">
-    <value>Bibliotheken auf Startseite anzeigen</value>
-  </data>
-  <data name="SettingsWidgetsTitle.Text" xml:space="preserve">
-    <value>Widgets</value>
-  </data>
-  <data name="SettingsWidgets.Content" xml:space="preserve">
-    <value>Widgets</value>
-  </data>
-  <data name="SettingsWidgetsShowDrives.AutomationProperties.Name" xml:space="preserve">
-    <value>Laufwerke auf Startseite anzeigen</value>
-  </data>
-  <data name="SettingsWidgetsShowDrives.Header" xml:space="preserve">
-    <value>Laufwerke auf Startseite anzeigen</value>
-  </data>
-  <data name="SettingsWidgetsShowRecentFiles.AutomationProperties.Name" xml:space="preserve">
-    <value>Zuletzt verwendete Dateien auf Startseite anzeigen</value>
-  </data>
-  <data name="SettingsWidgetsShowRecentFiles.Header" xml:space="preserve">
-    <value>Zuletzt verwendete Dateien auf Startseite anzeigen</value>
-  </data>
-  <data name="SettingsPreferencesCopyLocationSwitch.Header" xml:space="preserve">
-    <value>Eine Option anzeigen, um den Speicherort des ausgewählten Elements zu kopieren</value>
-  </data>
-  <data name="SettingsFilesAndFoldersShowHiddenItems.Header" xml:space="preserve">
-    <value>Versteckete Dateien und Ordner anzeigen</value>
-  </data>
-  <data name="SettingsFilesAndFoldersShowHiddenItems.AutomationProperties.Name" xml:space="preserve">
-    <value>Versteckete Dateien und Ordner anzeigen</value>
-  </data>
-  <data name="EjectNotificationHeader" xml:space="preserve">
-    <value>Hardware sicher entfernen</value>
-  </data>
-  <data name="EjectNotificationBody" xml:space="preserve">
-    <value>Das Gerät kann jetzt sicher vom PC entfernt werden.</value>
-  </data>
-  <data name="EjectNotificationErrorDialogHeader" xml:space="preserve">
-    <value>Problem beim auswerfen des Gerätes</value>
-  </data>
-  <data name="EjectNotificationErrorDialogBody" xml:space="preserve">
-    <value>Dieses Gerät wird derzeit benutzt. Schließen Sie jedes Program, Fenster oder Tab das möglicherweise das Gerät benutzt und versuchen Sie es erneut.</value>
-  </data>
-  <data name="SideBarEjectDevice.Text" xml:space="preserve">
-    <value>Auswerfen</value>
-  </data>
-  <data name="HorizontalMultitaskingControlDuplicateTab.Text" xml:space="preserve">
-    <value>Tab duplizieren</value>
-  </data>
-  <data name="HorizontalMultitaskingControlMoveTabToNewWindow.Text" xml:space="preserve">
-    <value>Tab in neuem Fenster öffnen</value>
-  </data>
-  <data name="HorizontalMultitaskingControlNewTab.Text" xml:space="preserve">
-    <value>Neuer Tab</value>
-  </data>
```

Comment:
@winston-de Here is an exmaple, it looks like some of the translated strings were deleted.

Reply:
@yaichenbaum Thanks. I think I messed up the translation resource files when merging.
Anyway, it should be fixed now.